app/src/main/res/layout/activity_review.xml

```diff
@@ -15,26 +15,29 @@
 
             <include layout="@layout/toolbar" />
 
+            <Button
+                android:id="@+id/skip_image"
+                android:layout_width="match_parent"
+                android:layout_height="25dp"
+                android:background="@android:color/transparent"
+                android:layout_below="@+id/toolbar"
+                android:text="SKIP THIS IMAGE"
```

Comment:
I know it wasn't previously, but it should use a `@string` resource rather than hardcoded text.